jenkins/release.jenkinsFile

```diff
@@ -14,4 +14,11 @@ standardReleasePipelineWithGenericTrigger(
             mavenArtifactsPath: "$WORKSPACE/repository/",
             autoPublish: true
         )
+        publishToArtifactsProdBucket(
+            assumedRoleName: 'sql-jdbc-upload-role',
```

Comment:
I would need to create this role. Will get started and let you know once that is done!

Reply:
Thanks so much!

Reply:
Hey! I am taking over from @rishabh6788 to integrate jar signer as well. So will get both of this done in one go. Will comment on this issue once it is done. Expected Thursday EOD. 
Thanks!

Reply:
The role has been created. Resolving the comment.